src/views/Products.vue

```diff
@@ -6,6 +6,18 @@
           <div class="text-center">
             <h1>Products table</h1>
           </div>
+
+          <div class="d-flex justify-end">
+            <v-btn
+              class="mx-4"
+              rounded="xs"
+              :size="isSmAndDown ? 'small' : 'x-large'"
```

Comment:
I've renamed the isSmAndDown to isSmallScreen. I think it makes more sense. What does SmAndDown mean?